### PR TITLE
Upgrade from ZEIT Now v1 to Vercel

### LINKED ISSUE #266: Site will no longer work after August 7th, 2020.
While exploring `now` locally, I received the following deprecation warnings:

> Error! As of June 19th 2020, the creation of new ZEIT Now 1.0 Deployments is prohibited. Existing Deployments will continue to be available until August 7th 2020.
>Learn More: https://vercel.link/v1-shutdown-announcement

When following that link, it goes into more detail:

> 7th of August 2020: All ZEIT Now 1.0 deployments will be deleted.

I assume this means the website will be down entirely without intervention.

### PR DESCRIPTION
**Pull request by Timer**
This upgrades from the deprecated ZEIT Now v1 platform to the Vercel ("v2") platform. This fixes the brownouts the site is currently experiencing.

p.s. I'd be happy to help with maintenance of this repo if you add me as a collaborator (or admin so I can fix up the CI)!

---

Closes #266

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ app.use((err, req, res) => {
   res.status(err.status || 500);
   res.render("error", {
     message: err.message,
-    error: {}
+    error: {},
```
Comment (ai) [inline app.js:52]: Can you remove code style changes?

File: vercel.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "version": 2,
+  "env": {
+    "GA_ID": "UA-543420-21"
```
Comment (ai) [inline vercel.json:4]: Let’s remove Google Analytics since it is a privacy bad practice

File: app.js (hunk 1 above)
Comment (Timer) [reply app.js:52]: These style changes were required by the linter. Please see the failing CI on prior commit.

File: vercel.json (hunk 2 above)
Comment (Timer) [reply vercel.json:4]: Sure! I didn't want to edit existing code (this tracking already existed), but I can remove this.

Comment (Timer) [reply vercel.json]: Removed GA.

**Comment by ai** [comment]
Let’s use Vercel as a short-term solution to fix an upcoming problem.

But I see a bigger picture here. To fix a long-term issue with website we need to make redesign and show warnings about bad Browserslist queries. 

Talking about a long-term solution for Browserslist website hosting. Does Vercel collect data about visitors?

File: app.js (hunk 1 above)
Comment (ai) [reply app.js:52]: Oh. Thanks. I didn’t updater linter config yet (spend too much time on preparing PostCSS 8 😓).

**Comment by Timer** [comment]
You can find our [Privacy Policy here](https://vercel.com/legal/privacy-policy), which should cover any questions you have!

**Comment by ai** [comment]
@Timer so Vercel will collect visitor information?

In this case, I will accept this PR, but when I finish PostCSS 8 I will try to find some simple Docker hosting with privacy attention (seems like Europe startups is an only hope).

**Comment by Timer** [comment]
IANAL so I can't interpret the document. However, I'd like to mention we're fully GDPR compliant and collect the minimum amount of information necessary for legal compliance in the jurisdictions we operate.

**Comment by Timer** [comment]
FYI this is the current live site, the first 30 minutes of every hour:
image

This PR will fix this error!

**Comment by ai** [comment]
> This PR will fix this error!

Yeap. I am OK with a short-term solution since we need to help users first and then think about privacy. Do I need just to accept PR or maybe I need to configure something in Vercel?

I just thought that you as a Vercel engineer discussed privacy questions with your team. I will investigate the privacy question laters.

**Comment by ai** [comment]
Hm. Seems like PR is blocked by the `Now` task.

I will ask @jonrohan for Admin permission.

**Comment by Timer** [comment]
Switching the required task from "now" to "Vercel" and then merging this PR should be all that's necessary.

That should re-enable the auto-upgrades of `caniuse-db` as well.

**Comment by ai** [comment]
I merged and added website to new Vercel team

**Comment by ai** [comment]
@Timer I haven’t access to old Now account. Hope your PR is enough.

**Comment by Timer** [comment]
Hmm, it looks like the `master` commit tried to deploy to an account with the following slug: `browserslistsite` instead of the personal account that was being deployed to on this PR (`jonrohan`).

It looks like someone re-connected this Git repository to a different account, thus messing up the deployment integration. This would've worked if the `browserslistsite` change wasn't made, so I think we need to wait for `jonrohan` to re-connect this repo to his account.

**Comment by Timer** [comment]
Who owns the `browsersl.ist` and `browserl.ist` domains?

**Comment by ai** [comment]
@Timer I have access to domains. The old connection was lost because of the repo transfer.

Seems like I need to ask @jonrohan to delete old Now account and then I need to spend time in creating a new Vercel account and binding it to new domains.

**Comment by Timer** [comment]
Ah, I just saw this repo was moved to a different org. @ai I'd be happy to help you setup the new account!

Did you create a team on Vercel? Or do you want this deployed on your personal account?

Either way, you'll want to go through our Import flow:
https://vercel.com/import/git

**Comment by ai** [comment]
@Timer can I use free Vercel account to work with Teams? Or Teams can be used only by Pro account?

> Personal accounts on Vercel can only be connected to personal accounts on GitHub, not organizations.

**Comment by Timer** [comment]
@ai a team is required to connect to a GitHub organization, yes. We sponsor OSS teams so it is not necessary for you to upgrade to Pro!

Once you create a team with a two week trial you can email `support@vercel.com` and have a sponsorship applied.

**Comment by ai** [comment]
> Once you create a team with a two week trial you can email support@vercel.com and have a sponsorship applied.

Can you give me a text to email?

**Comment by ai** [comment]
@Timer what setting I should set in import wizard?

**Comment by Timer** [comment]
Those default settings look appropriate!

**Comment by Timer** [comment]
I've pinged someone on our team to comment here about the OSS sponsorship!

**Comment by paulogdm** [comment]
Hello,

Here is the process to apply for sponsorship: https://vercel.com/knowledge/can-vercel-sponsor-my-project

We can make a few PRs to make the site optimized for the Vercel environment (caching and etc.)

Let me know if you want to apply for the sponsorship so I can keep an eye out on my inbox.

**Comment by ai** [comment]
@paulogdm maybe you can answer and privacy questions and Vercel.

Do you collect a privacy data of website visitors and sell this data to other companies?

**Comment by paulogdm** [comment]
We do not sell data to other companies in any way. If we did, I would quit on the spot.

Also, it is unlawful to do so as far as I'm aware. We only keep logs for administrative and moderation purposes (WAF, traffic monitoring, edge capacity, etc.).

**Comment by ai** [comment]
@Timer what DNS nameservers I should add to the domain to connect it with Vercel DNS?

**Comment by ai** [comment]
@paulogdm I worry a little because I see different statements in [Vercel’s Privacy Policy](https://vercel.com/legal/privacy-policy).

> When you download, use or interact with the Site …
> Information About your Device: information about the devices and software you use to access the Site — primarily the internet browser or mobile device that you use, the website or source that linked or referred you to the Site, your IP address or device ID (or other persistent identifier that uniquely identifies your computer or mobile device on the Internet), the **operating system of your computer or mobile device, device screen size,** and other similar technical information.

Seems like it is a lot of information just for “administrative and moderation purposes (WAF, traffic monitoring, edge capacity, etc.)”.

> We, or authorized third parties, automatically collect log data when you access and use the Platform, even if you have not created an account or logged in. We use this information to administer and improve the Services, analyze trends, track users' use of the Platform, and gather broad demographic information for aggregate use.
> Note that Device and Usage Information is collected from our community boards, as well as from our Customers' End Users who visit our Customers' websites hosted by Vercel.

Here is a Third Parties mention and “analyze trends, track users' use of the Platform, and gather broad demographic information for aggregate use” can be enough to sell user’s data to Ads networks.

For instance,  think that even Google Analytics usage is bad fr user privacy because Google uses GA data in Ads Network.

**Comment by paulogdm** [comment]
Hello @ai,

I can't really have a say on the policy since it is out of my area of expertise. However, I do think you should be as comfortable as possible with your hosting options. If Vercel is not a good fit for the project, I can totally understand.

If you have any questions, we are here.

**Comment by ai** [comment]
@Timer how I should add a custom domain to the website? In docs, I found only the “Transfer domain” feature.

**Comment by paulogdm** [comment]
@ai In regards to the privacy policy that you mentioned, it is specific to the vercel.com scope. Therefore, this **does not apply** to hosted sites. Take a look at the first paragraph of the policy.

**Comment by Timer** [comment]
@ai you should be able to add the domain to the "Domains" tab in project Settings, which will then guide you through the domain verification process.

**Comment by ai** [comment]
> you should be able to add the domain to the "Domains" tab in project Settings, which will then guide you through the domain verification process.

Thanks for your help. Seems like everything is working now.

**Comment by Timer** [comment]
Seems https://browserl.ist/ isn't setup to redirect to https://browsersl.ist/, can you configure that in the UI?

**Comment by ai** [comment]
@Timer the only problem is with `https://browserl.ist/` and `https://browserSl.ist/` aliases.

What do I need to do to support both domains?

I added both domains and `ALIAS` record





But the domain is not working https://browserl.ist/

**Comment by Timer** [comment]
Did you add both domains in the project settings?

image

**Comment by ai** [comment]
> Did you add both domains in the project settings?

Yeap

**Comment by Timer** [comment]
You can configure the other domain to be a redirect:
image

Can you show me a screenshot of your domains?

**Comment by ai** [comment]
I can’t find this domain settings page

**Comment by Timer** [comment]
It's on the project itself!

**Comment by ai** [comment]
> It's on the project itself! That's the team settings.

Done! I thought that Domain tab was a Project’s tab, but only not I got that there project’s settings are in different place.

**Comment by Timer** [comment]
Seems to work for me now. Awesome!

**Comment by ai** [comment]
@paulogdm 

> In regards to the privacy policy that you mentioned, it is specific to the vercel.com scope. Therefore, this does not apply to hosted sites. Take a look at the first paragraph of the policy.

Yeap, it is confusing. It is not clear what rules are accepted to the Vercel admin page and what to hosted sites.

Where I can find Privacy Policies for hosted sites?

**Comment by paulogdm** [comment]
You can find it here: https://vercel.com/legal/privacy-policy#customers

Apologies for the confusion.

**Comment by Timer** [comment]
The privacy policy for hosted sites is this specific section:
https://vercel.com/legal/privacy-policy#customers

**Comment by ai** [comment]
If you need feedback, here are things that worry me in those sections

> This information may also **include, but is not limited**

No clear list of what is collected

> With respect to Log Data, we collect and use Log Data to operate, maintain, and improve our Services in performance of our obligations under our Customer agreement

It is too unclear how exactly my website visitor data will be used. No explicit prohibit of sending the data to third-parties. No description of how long Log Data will be stored.

Here is a good example of Privacy Policy: https://mullvad.net/en/help/privacy-policy/

> Your personal data will only be shared with third party suppliers who are performing services on our behalf and for the purposes stated above. The categories of such recipients are e-mail service providers and payment solution suppliers (which are subject to confidentiality).

Clear examples without include, but is not limited

> Categories of personal data

A clear list of what data is collected

> https://mullvad.net/ru/help/no-logging-data-policy/

Explicit no logging data policy

**Comment by paulogdm** [comment]
@ai Thank you very much! I have relayed it to the team.

**Pull request merged**